units/stats_queue.go

```diff
@@ -0,0 +1,74 @@
+package units
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/scheduler"
+	"github.com/mongodb/amboy"
+	"github.com/mongodb/amboy/dependency"
+	"github.com/mongodb/amboy/job"
+	"github.com/mongodb/amboy/registry"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/logging"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+)
+
+const queueStatsCollectorJobName = "queue-stats-collector"
+
+func init() {
+	registry.AddJobType(queueStatsCollectorJobName,
+		func() amboy.Job { return makeQueueStatsCollector() })
+}
+
+type queueStatsCollector struct {
+	job.Base `bson:"job_base" json:"job_base" yaml:"job_base"`
+	logger   grip.Journaler
```

Comment:
there should either be a test for this or this field should be removed (I'm fine with no test since this job is pretty simple)

Reply:
done